Source/inv.cpp

```diff
@@ -591,6 +591,14 @@ bool AutoEquip(int playerNumber, const ItemStruct &item, bool persistItem)
  */
 bool AutoEquipEnabled(const PlayerStruct &player, const ItemStruct &item)
 {
+	/*
+	// Use this when Gameplay.bAutoEquipXXX flags are removed.

```

Comment:
Please remove commented code. I also do not think the flags should be removed.

Reply:
The very purpose of this change is to remove these flags from `Diablo.ini` and `struct GamePlayOption` eventually.

Reply:
the very purpose of git is not having to leave commented code